### DISCUSSION TITLE
How to move the Windows 11 style Taskbar to the top?

### DISCUSSION THREAD

**Discussion by NanoAi** [valinet/ExplorerPatcher#1360] [Q&A] [answered]
Over at https://github.com/valinet/ExplorerPatcher/wiki/All-features#primarysecondary-taskbar-location-on-screen it says that the Windows 11 taskbar is able to be moved to the top using Explorer Patcher however I don't see any options to do so.

![image](https://user-images.githubusercontent.com/3096584/233808584-682b520f-9ca7-4aa1-be96-dd356e0316c7.png)

**Comment by valinet** [comment, marked as answer]
The Windows 11 taskbar could be moved to the top in 22000-based builds, but the functionality stopped working in 22621-based builds (22H2), so it is hidden from the Properties window when running 22H2: https://github.com/valinet/ExplorerPatcher/blob/master/ExplorerPatcher/settings.reg#L19

As for the docs, they are outdated on this one. I have corrected them now.

**Reply by granterogers** [reply]
Is there any plan to address this shortcoming if possible?

**Comment by kimjboland** [comment]
Does this mean it is not possible to have the taskbar on the left side of the screen with explorer patcher? 

If so, this is a shame (Shame on Microsoft) as this is the main reason I need ExplorerPatcher.

**Comment by Anixx** [comment]
Yiu can enable the Windows 10 - style taskbar, it can be put to any side of the screen.

**Comment by brewburgr** [comment]
how do you move the windows 10 taskbar to the top of the screen? I didn't find the option anywhere..

**Reply by Anixx** [reply]
Like you did since Windows 95: with mouse. Did you ever use Windows before?

**Reply by gex581990** [reply]
> Like you did since Windows 95: with mouse. Did you ever use Windows before?

You know that doesn't work anymore right? Lock the taskbar isn't even a thing anymore, if DWM screws up your taskbar size you can't even resize it anymore

**Reply by pyrates999** [reply]
it does if using the windows 10 taskbar.  Just right click on the taskbar and untick lock the taskbar.  Then you can move it.
